pom.xml

```diff
@@ -22,7 +22,6 @@
         <module>modules/GraphvizLayout</module>
         <module>modules/GraphStreaming</module>
         <module>modules/DesktopStreaming</module>
-        <module>modules/StreamingAPI</module>
```

Comment:
This must be an error. Also, it would be preferrable to make a pull request from a branch that only contains your plugin, instead of master-forge.